docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '3'
+services:
+  app:
+    user: ${MITODL_USER}
+    build: .
+    container_name: hugo-course-publisher
+    ports:
+      - "3000:3000"
+    volumes:
+      - ./app/site:/usr/share/app/site
+      - ./app/src:/usr/share/app/src
+      - ./app/dist:/usr/share/app/dist
+    command: npm start
```

Comment:
Can we replace `npm start` with `bash -c 'npm install && npm start'`? The npm install part should only be slow the first time or if there are changes in package dependencies.